### PR TITLE
Add Bootstrap and Font Awesome assets to third_party.

### LINKED ISSUE #3: Add Bootstrap and Font Awesome CSS and JS assets
The site currently does not render properly because the Bootstrap and Font Awesome CSS, JS, and font assets are missing.

In the bazelbuild/bazel repo, these assets live separately under `third_party`. @steren - do we need also need to keep these assets separate under a `third_party` directory in this repo as well or would it be fine to directly add them to the `assets/` directory?

### LINKED ISSUE #5: Rework BUILD and serve-docs script
Most of the logic for gathering the docs generated from source files can be removed since the marketing site is now completely static.

### PR DESCRIPTION
**Pull request by davidzchen**
* Add third_party directory and import bootstrap and font_awesome
  sources.
* Rename bazel-release.pub.gpg to bazel_release.pub.gpg due to
  bazelbuild/bazel#2909
* Add script and build step as workaround for bazelbuild/bazel#2909
  to rename bazel_release.gpg.pub  to bazel-release.gpg.pub when
  constructing jekyll-tree.tar archive.
* Add serve-docs.sh script with workaround for jekyll/jekyll#6060

Fixes #3
Fixes #5

### CODE REVIEW

**Comment by davidzchen** [comment]
To confirm, are we fine with renaming bazel-release.pub.gpg to bazel_release.pub.gpg? I did not find any links to the public key file in the website files but do we need to update any links elsewhere?

**Comment by damienmg** [comment]
There is a link to that file in the installation manual. There is no requirements for this file to have any name so we are fine with it.

File: BUILD [hunk 1]
```diff
@@ -1,36 +1,23 @@
...
-            "command-line-reference-prefix.html",
-            "command-line-reference-suffix.html",
             "site/README.md",
-        ] + DOT_GRAPH_HTML_FILES,
+            "LICENSE",
+            "CONTRIBUTING",
+            "production/**",
+            "README.md",
+        ],
     ),
```
Comment (damienmg) [inline BUILD]: To keep bazel-release you can add specifically the file bazel-release, instead of globbing it.

Comment (davidzchen) [inline BUILD]: Tried renaming back to bazel-release and using the following `filegroup` rule:

```python
filegroup(
    name = "jekyll-srcs",
    srcs = [
        "bazel-release.pub.gpg",
    ] + glob(
        ["**/*"],
        exclude = [
            ".git/**",
            "bazel-*/**",
            "BUILD",
            "WORKSPACE",
            "jekyll-tree.sh",
            "scripts/**",
            "*.swp",
            "site/README.md",
            "LICENSE",
            "CONTRIBUTING",
            "production/**",
            "README.md",
        ],
    ),
)
```

But still getting the same error seen in bazelbuild/bazel#2909:

```
❯❯ scripts/serve-docs.sh
expr: syntax error
INFO: Analysed target //:jekyll-tree.tar (9 packages loaded).
INFO: Found 1 target...
ERROR: missing input file '//:bazel-release.pub.gpg'
ERROR: /Volumes/Ocean/Projects/bazelbuild/bazel-website/BUILD:26:1: //:jekyll-files: missing input file '//:bazel-release.pub.gpg'
Target //:jekyll-tree.tar failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /Volumes/Ocean/Projects/bazelbuild/bazel-website/BUILD:26:1 1 input file(s) do not exist
INFO: Elapsed time: 3.943s, Critical Path: 0.01s
FAILED: Build did NOT complete successfully
```

Comment (damienmg) [inline BUILD]: Can I get a commit to repro on my machine?

Comment (davidzchen) [inline BUILD]: Yes, you can clone my fork and check out the `third-party` branch:

```sh
git clone https://github.com/davidzchen/bazel-website.git
cd bazel-website; git checkout third-party
```

Comment (davidzchen) [inline BUILD]: I have added a workaround using a script to rename `bazel_release.gpg.pub` to `bazel-release.gpg.pub` to unblock this and added a TODO to remove it once bazelbuild/bazel#2909 is fixed.

PTAL

Comment (steren) [inline BUILD]: ping @damienmg

Comment (damienmg) [inline BUILD]: I haven't forgot it, just hadn't got the time to get around it yet.

**Pull request merged**